File: C#/FrameworksPerformance/SolrEngine/Models/Product.cs
using SolrNet.Attributes;

namespace SolrEngine.Models;

internal class Product
{

    [SolrUniqueKey("id")]
    public int Id { get; set; }
    [SolrField("name")]
    public string Name { get; set; }
    [SolrField("description")]
    public string Description { get; set; }
    [SolrField("price")] 
    public double Price { get; set; }
    [SolrField("category")]
    public string Category { get; set; }
}

File: Java/JavaDatabaseFrameworks/SolrApi/src/main/java/org/dbFrameworks/entities/Product.java
package org.dbFrameworks.entities;

import org.apache.solr.client.solrj.beans.Field;

public class Product {
    public Product(String id, String name, String description, double price, String category) {
        this.id = id;
        this.name = name;
        this.description = description;
        this.price = price;
        this.category = category;
    }
    public Product() {}
    @Field("id") public String id;
    @Field("name") public String name;
    @Field("description")public String description;
    @Field("price") public double price;
    @Field("category") public String category;

}
